data.json

```diff
@@ -35,6 +35,15 @@
     "dateAdded": "2018-12-09T07:08:00.000Z",
     "tags": ["stickers"]
   },
+  {
+    "name": "Appwrite",
+    "difficulty": "medium",
+    "description": "Make pull requests to issues tagged <code>Hacktoberfest</code> or <code>help wanted</code> in the <a href='https://github.com/appwrite/appwrite'>appwrite repo</a> during October to get some stickers and badges.",
+    "reference": "https://medium.com/appwrite-io/hacktoberfest-2019-is-almost-here-lets-celebrate-it-together-24b311236dd",
+    "image": "https://miro.medium.com/max/1600/1*mk5pBbuAZUg9j7nc6lbfrg.jpeg",
+    "dateAdded": "2019-10-06T20:30:00.000Z",
+    "tags": ["hacktoberfest", "stickers", "badges"]
```

Comment:
```suggestion
    "tags": ["badges", "hacktoberfest", "stickers"]
```

Reply:
I was going to make all the changes at once later, but we can make them here too 👍